### ISSUE TITLE
mode slowdown cursor movement

### ISSUE THREAD

**Issue by 11111000000** [ema2159/centaur-tabs#200]
it slowdown cursor movement in buffer

**Comment by jcs090218** [comment]
Can you elaborate more on the problem? This package has no impact on the cursor movement. 🤔

**Comment by jcs090218** [comment]
Yeah, the project finding is very slow with the Tramp. Projectile has a similar issue; see https://github.com/bbatsov/projectile/issues/1232.
